app/v2/services/AmendForeignPropertyPeriodSummaryService.scala

```diff
@@ -16,34 +16,25 @@
 
 package v2.services
 
-import api.controllers.EndpointLogContext
+import api.controllers.RequestContext
 import api.models.errors._
-import api.services.ServiceOutcome
-import api.support.DownstreamResponseMappingSupport
-import cats.data.EitherT
+import api.services.{BaseService, ServiceOutcome}
 import cats.implicits._
-import uk.gov.hmrc.http.HeaderCarrier
-import utils.Logging
 import v2.connectors.AmendForeignPropertyPeriodSummaryConnector
 import v2.models.request.amendForeignPropertyPeriodSummary.AmendForeignPropertyPeriodSummaryRequest
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class AmendForeignPropertyPeriodSummaryService @Inject() (connector: AmendForeignPropertyPeriodSummaryConnector)
-    extends DownstreamResponseMappingSupport
-    with Logging {
+class AmendForeignPropertyPeriodSummaryService @Inject() (connector: AmendForeignPropertyPeriodSummaryConnector) extends BaseService {
 
   def amendForeignPropertyPeriodSummary(request: AmendForeignPropertyPeriodSummaryRequest)(implicit
-      hc: HeaderCarrier,
-      ec: ExecutionContext,
-      logContext: EndpointLogContext,
-      correlationId: String): Future[ServiceOutcome[Unit]] = {
+      ctx: RequestContext,
+      ec: ExecutionContext
+  ): Future[ServiceOutcome[Unit]] = {
```

Comment:
For the outcomes can we add a type into the v2/package.scala like [here](https://github.com/hmrc/property-business-api/blob/main/app/v2/services/package.scala) and use it as the outcome like is done [here](https://github.com/hmrc/property-business-api/blob/main/app/v2/services/AmendHistoricNonFhlUkPiePeriodSummaryService.scala)